### PR TITLE
adjust the layout for the jsbrowser error feedback ui

### PR DESCRIPTION
**Pull request by devoncarew**
- adjust the layout for the jsbrowser error feedback ui
- remove one or two null checks that are now no longer needed (since we've propagated our nullability annotations a bit further through the project)

This breaks the error feedback UI from a single long line into two shorter ones; this is to make it work better in (generally fairly narrow) tool windows.

Screen Shot 2020-10-04 at 3 55 08 PM

### CODE REVIEW

**Comment by devoncarew** [comment]
Helin - this PR isn't timely, just some useful UX changes. I fixed two test failures introduced by this PR but am having some trouble w/ the 3rd:

```
Wanted but not invoked:
devToolsManager.openBrowserIntoPanel(
    "http://www.testUrl.com",
    null,
    "Test Project Name",
    "inspector"
);
-> at io.flutter.view.FlutterViewTest.testHandleJxBrowserInstalledWithDevtoolsSucceeded(FlutterViewTest.java:101)

However, there were exactly 2 interactions with this mock:
devToolsManager.hasInstalledDevTools();
-> at io.flutter.view.FlutterView.presentDevTools(FlutterView.java:392)

devToolsManager.installDevTools();
-> at io.flutter.view.FlutterView.presentDevTools(FlutterView.java:398)
```

File: src/io/flutter/view/FlutterView.java [hunk 1]
```diff
@@ -199,18 +200,12 @@ private void addBrowserInspectorViewContent(FlutterApp app,
...
-      tabName = app.getProject().getName();
-    }
-    else {
-      final List<FlutterDevice> existingDevices = new ArrayList<>();
-      for (FlutterApp otherApp : perAppViewState.keySet()) {
-        existingDevices.add(otherApp.device());
-      }
-      tabName = device.getUniqueName(existingDevices);
+    final List<FlutterDevice> existingDevices = new ArrayList<>();
+    for (FlutterApp otherApp : perAppViewState.keySet()) {
```
Comment (helin24) [inline src/io/flutter/view/FlutterView.java:205]: I'm seeing a null pointer exception here when debugging the test that failed, it looks like during the test `perAppViewState` is null. Is this likely to only happen in the test environment? Maybe we could put this retrieving of a unique name into a helper function.

Comment (devoncarew) [reply src/io/flutter/view/FlutterView.java:205]: I think that the `device` should be non-null for all normal invocations (normal app runs) but can be null for tests. I'll likely back out removing the null checks - it's not a critical back of this PR.

**Review by helin24 — APPROVED** [review]
Looks nice!

**Pull request merged**